addons/goldpinger/template/yaml/troubleshoot.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: v1
```

Comment:
The testgrid tests are passing, but for some reason I cannot figure out how this `troubleshoot.yaml` spec is applied to the cluster when installing the add-on!

Reply:
I can guarantee that it's not being applied if it's only here in the template directory

Reply:
The file is in `template/yaml`.

I have specific tests checking that the secret gets created and they are passing. It seems the secret gets applied, I just can't tell where that happens

Reply:
No, I mean that the template directory is only used to generate the actual versioned addons - it's not included in the things being run on testgrid.

And `template/yaml` isn't a directory used by anything to the best of my knowledge - if you want this to be included in future versions it would need to be added to https://github.com/replicatedhq/kURL/tree/main/addons/goldpinger/template/base instead